src/css/global.css

```diff
@@ -1,6 +1,7 @@
 :root {
     /* Root font-size = 10px / 0.1rem = 1px */
     font-size: 62.5%;
+    font-size-adjust: 100%;
```

Comment:
What's the point of setting this if you don't ever change it?

Reply:
Some browsers have a shitty default that will change the x-height of fonts to make them much bigger, I think iOS does it (or used to). This just makes sure nothing weird happens.